### DISCUSSION TITLE
Current state of project

### DISCUSSION THREAD

**Discussion by traek** [ryanhowdy/fcms#617] [General]
It looks like the last work on the project was done exactly one year ago (great timing for me, right?). I was wondering if everyone just got burned out or got the 'rona or what...

Is this project still alive? If so, what's the current timeframe on the [Next Release](https://github.com/ryanhowdy/fcms/projects/2)? I'm particularly interested in #390 mostly since I think it'd be a great way to collaborate on a shared tree and allow them to export what they want and continue their work offline.

**Comment by ryanhowdy** [comment]
It's alive, but just barely.

I don't get to work on it too often, but I am still working.  And because of that I have no idea about timelines.  I usually work on stuff when someone gets interested.

**Reply by ryanhowdy** [reply]
Please see this announcement for more information about the [future of Family Connecitons](https://github.com/ryanhowdy/fcms/discussions/635).
